integration/daemon/daemon_test.go

```diff
@@ -551,6 +552,8 @@ func testLiveRestoreVolumeReferences(t *testing.T) {
 			cID := container.Run(ctx, t, c, container.WithMount(m), container.WithCmd("top"), container.WithRestartPolicy(policy))
 			defer c.ContainerRemove(ctx, cID, containertypes.RemoveOptions{Force: true})
 
+			poll.WaitOn(t, container.IsInState(ctx, c, cID, "running"), poll.WithDelay(100*time.Millisecond))
+
```

Comment:
Same for this one (and other ones that check for `running` state after calling `container.Run`